src/dashboard/components/button.py

```diff
@@ -0,0 +1,47 @@
+"""Button component."""
+from typing import Any
+
+from dash import html
+
+from dashboard.components.icon import icon
+
+
+def button(
+    icon_name: str,
+    text: str,
+    fill: int = 0,
+    weight: int = 400,
+    grade: int = 0,
+    size: int = 16,
+    **kwargs: Any,
+) -> html.Button:
+    """Create a general button containing an icon to the left.
+
+    Args:
+        icon_name (str): the name of the icon, for more information see:
+            https://fonts.google.com/icons?icon.set=Material+Symbols
+        text (str): The text to be displayed inside the button.
+        fill (int): Set to 0 for outlined, 1 for filled. Default 0.
+        weight (int): Font weight for the icon, range 100-700,
+            default 400.
+        grade (int): Affects font weight, range -25-200, default 0.
+        size (int, optional): The font size in pixels, also sets icon
+        size. Defaults to 16
+        kwargs (Any): Forwarded to html.Button.
+
+    Returns:
+        html.Button: The button.
+    """
+    class_name = kwargs.pop("className", "")
+
+    return html.Button(
+        **kwargs,
```

Comment:
You need to handle the case where a className is included in the `kwargs` dict. See icon implementation.

Reply:
Is it not handled with kwargs.pop already, like the icon implementation?

Reply:
Yes, I wrote this comment before seeing the changes made in 7d66383.